From 86dfab1517d3417e3063812b1fe5ad10760a0bab Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@samba.org>
Date: Wed, 15 Jul 2015 13:22:40 +0200
Subject: [PATCH] docs: Documents length limitations for NetBIOS name

BUG: https://bugzilla.samba.org/show_bug.cgi?id=11401

Signed-off-by: Andreas Schneider <asn@samba.org>
Reviewed-by: Michael Adam <obnox@samba.org>

Autobuild-User(master): Andreas Schneider <asn@cryptomilk.org>
Autobuild-Date(master): Wed Jul 15 19:35:48 CEST 2015 on sn-devel-104

(cherry picked from commit e736a169d5dc76d0ef10d20f211ea6e7e6df4f59)
---
 docs-xml/smbdotconf/base/netbiosname.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/docs-xml/smbdotconf/base/netbiosname.xml b/docs-xml/smbdotconf/base/netbiosname.xml
index 60bc082..542d058 100644
--- a/docs-xml/smbdotconf/base/netbiosname.xml
+++ b/docs-xml/smbdotconf/base/netbiosname.xml
@@ -11,6 +11,8 @@
 		the hosts DNS name) will be the name that these services are advertised under.
 		</para>
 
+		<para>Note that the maximum length for a NetBIOS name is 15 charactars.</para>
+
 		<para>
 		There is a bug in Samba that breaks operation of browsing and access to shares if the netbios name
 		is set to the literal name <literal>PIPE</literal>. To avoid this problem, do not name your Samba
-- 
2.4.5

